exercises/react/tests/react.rs

```diff
@@ -0,0 +1,14 @@
+extern crate react;
+
+#[allow(unused_mut)]
+
+// TODO: [ignore] tests
+
+#[test]
+fn set_value_of_input() {
+    let mut reactor = react::Reactor::new();
+    let mut input = reactor.create_input(1);
+    assert_eq!(input.value(), 1);
+    input.set_value(2);
```

Comment:
From what I read of https://www.reddit.com/r/rust/comments/2uvfic/why_doesnt_rust_have_properties/, this is *slightly* un-Rusty, but one point made is "Don't add ways to access/modify data unless it's absolutely necessary." and I believe in this case it *is* necessary since when a value is set the cell needs to propagate the value to any dependents. This isn't possible if the interface is a simple `input.value = 2`